WP_Auth0.php

```diff
@@ -693,6 +691,31 @@ function wp_auth0_ajax_rotate_migration_token() {
 }
 add_action( 'wp_ajax_auth0_rotate_migration_token', 'wp_auth0_ajax_rotate_migration_token' );
 
+/**
+ * AJAX handler to re-send verification email.
+ * Hooked to: wp_ajax_nopriv_resend_verification_email
+ *
+ * @codeCoverageIgnore - Tested in TestEmailVerification::testResendVerificationEmail()
+ */
+function wp_auth0_ajax_resend_verification_email() {
```

Comment:
Moved here from `wp_auth0_ajax_resend_verification_email()` below (same function) and merged with `WP_Auth0_Email_Verification::resend_verification_email()`